### ISSUE TITLE
Changing sass variables

### ISSUE THREAD

**Issue by petermanders89** [josdejong/jsoneditor#881]
I noticed that in the SCSS files the file styles is imported all over. This file contains variables, however it is not easy to overwrite them. Since the menu will still be blue:

```
$jse-blue: red;

@import "jsoneditor/src/scss/menu";
```

I am wondering why the variables within the styles.scss are not set with !default. With !default you can easily overwrite the styling in custom applications. Other libraries, like bootstrap for example, have most of their variables defined with the !default parameter. It would make overwriting the styling easier and friendly. The following example will still have a red color:

```
$jse-blue: red;
$jse-blue: blue !default;

@import "jsoneditor/src/scss/menu";
```

Not sure if I should re-open an separate issue for this, but if the !default parameter is added, I would suggest changing the name of the variable `$jse-blue` to something more generic.

**Comment by josdejong** [comment]
Thanks for your suggestion. I'm all in for solutions who make it easier to customize the styling :)

I hadn't heard of `:default` before. Sounds good. The [page at MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/:default) suggests that Edge and IE don't support `:default`. Is this usable then? Or does SASS resolve this?

**Comment by petermandersvin** [comment]
The MDN is referring to an actual css selector, which is a different default. SASS takes care of this. How it works in SASS, the !default will only activate if the variable is not already defined. SASS does take care of this. A simple example:

```
$jse-blue: red;
$jse-blue: blue !default;

#test{
  background-color: $jse-blue;
}
```

After compiling to CSS it will turn into:

```
#test {
  background-color: red;
}
```

**Comment by josdejong** [comment]
Ah I understand it. Sounds good!

Anyone interested in implementing this improvement?

**Comment by josdejong** [comment]
This is now available in `v8.2.0`. Dank je wel Peter :)
